cukinia

```diff
@@ -25,6 +25,11 @@ GENERIC_FUNCTS='cmd
 # By default log to stdout
 __log_file="/dev/stdout"
 
+# Default temporary files
+__stderr_tmp=$(mktemp)
+__stdout_tmp=$(mktemp)
+__logfile_tmp=$(mktemp)
```

Comment:
that's better, I would add a trap 0 function to cleanup the temporary files at exit (voluntary or not)

```
cleanup()
{
    rm -f $__stderr_tmp $__stdout_tmp $__logfile_tmp
}

trap cleanup 0
```

Reply:
done